ambry-api/src/main/java/com.github.ambry/clustermap/ClusterMap.java

```diff
@@ -51,6 +51,20 @@
   boolean hasDatacenter(String datacenterName);
 
   /**
+   * Gets the id for the given datacenter name, which is unique across the entire cluster.
+   * @param datacenterName The name of the datacenter to get the id.
+   * @return The id of the datacenter.
+   */
+  Short getDatacenterIdByName(String datacenterName);
+
+  /**
+   * Gets the name for the given datacenter id, which is unique across the entire cluster.
+   * @param datacenterId The id of the datacenter to get the name.
+   * @return The name of the datacenter.
+   */
+  String getDatacenterNameById(short datacenterId);
```

Comment:
Whatever you expect to do with the name once you get it from the id - can't you do it with the id in the first place?